OpenOversight/app/templates/officer.html

```diff
@@ -186,6 +186,33 @@ <h4>
       	{% endif %}
     	</div>{# end col #}
     {% endif %}
+    <div class='col-sm-12 col-md-6'>
+       <h3>Descriptions</h3>
+       <ul class="list-group">
+          {% for description in officer.descriptions %}
+          <li class="list-group-item">
+                <em>{{ description.date_updated.strftime('%b %d, %Y')}}</em> <br/>
+                {{ description.text_contents }} <br/>
+                <em>{{ description.creator.username }} </em>
+             {% if description.creator_id == current_user.id or current_user.is_administrator %}
+                <a href="{{ url_for('main.description_api', officer_id=officer.id,
+                         obj_id=description.id) + '/edit' }}">
+                <span class='sr-only'>Edit</span>
+                <i class="fa fa-pencil-square-o" aria-hidden="true"></i>
+                </a>
+                <a href="{{ url_for('main.description_api', officer_id=officer.id,
+                         obj_id=description.id) + '/delete' }}">
+                <span class='sr-only'>Delete</span>
+                <i class="fa fa-trash-o" aria-hidden="true"></i>
+                </a>
+             </a>
+             {% endif %}
+          </li>
+          {% endfor %}
+       </ul>
+       <a href="{{ url_for('main.description_api', officer_id=officer.id) }}" class='btn btn-primary'>New description</a>
```

Comment:
We should only show this button if the user has rights to add the description, i.e. if they are an administrator or they are the area coordinator of this department: `{% if current_user.is_administrator or (current_user.is_area_coordinator and current_user.ac_department_id == officer.department_id) %}`